src/pages/find-offers/FindOffers.tsx

```
@@ -1,7 +1,59 @@
+    fetchOnMount: false
+  })
+
+  useEffect(() => {
+    if (!hasFetched) {
+      fetchData();
+      setHasFetched(true);
+    }
+  }, [hasFetched, fetchData]);
+
+  if (loading) {
+    return <Loader />;
```

ESLint

[error] Delete `;`

(prettier/prettier)